web/.storybook/preview.ts

```diff
@@ -1,6 +1,11 @@
 import type { Preview } from "@storybook/web-components";
 
+import "@goauthentik/common/styles/authentik.css";
```

Comment:
Yes!  As I quoted from that engineer at Google's Lit-Element dev team, Patternfly is _really_ not a good fit for Web Components because of Patternfly's "modules" are too big, resulting in heavy bloat of the constructed stylesheets. There is a technique in which a parent class holds _all_ the stylesheets at runtime, and children request them to assemble a component's styles, but that's both FLOUT-y and really difficult to manage.  The recommended alternative (even from Patternfly themselves!) is to build your own library out of Patternfly components.